client/src/components/modals_and_popovers/FixedPopover.stories.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import { FixedPopover } from "./FixedPopover.js";
+
+export default {
+  title: "FixedPopover",
+  component: FixedPopover,
+};
+
+const Template = (args) => <FixedPopover {...args} />;
```

Comment:
To make the "Fixed" aspect clearer, should wrap this in a tall container so that the story page can scroll.

Reply:
Done!